docs/content/doc/installation/database-preparation.en-us.md

```diff
@@ -154,3 +154,136 @@ Note: All steps below requires that the database engine of your choice is instal
     where `gitea` is database user, `giteadb` is database name, and `203.0.113.3` is IP address of your database instance.
 
     You should be prompted to enter password for the database user, and connected to the database.
+
+## Database Connection over TLS
+
+If you have database instance on private network, this section can be omitted, since the security between Gitea and database instance is guaranteed. If instead the database instance is on public network, use TLS to encrypt connection to database, as it is possible for third-party to intercept database connection.
```

Comment:
```suggestion
If the communication between Gitea and your database instance is performed through a private network, or if Gitea and the database are running on the same server, this section can be omitted since the security between Gitea and the database instance is not critically exposed. If instead the database instance is on a public network, use TLS to encrypt the connection to the database, as it is possible for third-parties to intercept the traffic data.
```